locale/retext_tr.ts

```diff
@@ -0,0 +1,849 @@
+        <source>Disabled</source>
+        <translation>Pasif</translation>
+    </message>
+    <message>
+        <location filename="../ReText/config.py" line="192"/>
+        <source>Cursor Line</source>
+        <translation>İmleç Çizgisi</translation>
+    </message>
+    <message>
+        <location filename="../ReText/config.py" line="193"/>
+        <source>Wrapped Line</source>
+        <translation>Sarılmış Çizgi</translation>
+    </message>
+    <message>
+        <location filename="../ReText/config.py" line="198"/>
+        <source>Increment</source>
+        <translation>Artır</translation>
+    </message>
+    <message>
+        <location filename="../ReText/config.py" line="199"/>
+        <source>Repeat</source>
+        <translation>Yinele</translation>
+    </message>
+</context>
+<context>
+    <name>DirectorySelectButton</name>
+    <message>
+        <location filename="../ReText/config.py" line="60"/>
+        <source>Select directory to open</source>
+        <translation>Açılacak dizini seçin</translation>
+    </message>
+</context>
+<context>
+    <name>FileDialogButton</name>
+    <message>
+        <location filename="../ReText/config.py" line="35"/>
+        <source>(none)</source>
+        <translation>(yok)</translation>
+    </message>
+</context>
+<context>
+    <name>FileSelectButton</name>
+    <message>
+        <location filename="../ReText/config.py" line="52"/>
+        <source>Select file to open</source>
+        <translation>Açılacak dosyayı seçin</translation>
+    </message>
+</context>
+<context>
+    <name>InsertTableDialog</name>
+    <message>
+        <location filename="../ReText/tabledialog.py" line="27"/>
+        <source>Insert table</source>
+        <translation>Tablo ekle</translation>
+    </message>
+    <message>
+        <location filename="../ReText/tabledialog.py" line="36"/>
+        <source>Number of rows</source>
+        <translation>Satırların sayısı</translation>
+    </message>
+    <message>
+        <location filename="../ReText/tabledialog.py" line="37"/>
+        <source>Number of columns</source>
+        <translation>Sütunların sayısı</translation>
+    </message>
+</context>
+<context>
+    <name>LocaleDialog</name>
+    <message>
+        <location filename="../ReText/dialogs.py" line="44"/>
+        <source>Enter locale name (example: en_US)</source>
+        <translation>Yerel adı girin (örnek: en_US)</translation>
+    </message>
+    <message>
+        <location filename="../ReText/dialogs.py" line="50"/>
+        <source>Set as default</source>
+        <translation>Öntanımlı olarak ata</translation>
+    </message>
+</context>
+<context>
+    <name>Proxy</name>
+    <message>
+        <location filename="../ReText/fakevimeditor.py" line="61"/>
+        <source>No write since last change (add ! to override)</source>
+        <translation>Son değişiklikten bu yana yazma yok (geçersiz kılmak için! Ekleyin)</translation>
```

Comment:
Here the string means that one should literally type an exclamation mark. Did you translate it correctly? It is suspicious to me that `Ekleyin` is capitalized. And maybe add a space before `!`?